activerecord/CHANGELOG.md

```diff
@@ -1001,7 +995,8 @@
     Post.first.inspect #=> "#<Post id: 1, title: "Hello, World!", published_at: "2023-10-23 14:28:11 +0000">"
     ```
 
-    In `development` and `test` mode, `attributes_for_inspect` will be set to `:all` by default.
+    By default `attributes_for_inspect` is set to `:all`.
+    For new appliciations it is set to `[:id]` in `config/environment/production.rb`.
```

Comment:
Should the change in behavior be added to `Unreleased` at the top (since the current text is correct as of the release its a part of?)

Reply:
Ah yes, you are correct. I didn't know we had versioning in released CHANGELOGs.
I'll update it.